docs/server/mcp/tools/get-migration-guide.ts

```
@@ -3,32 +3,41 @@ import { queryCollection } from '@nuxt/content/server'
+    idempotentHint: true,
+    openWorldHint: false
+  },
   inputSchema: {
     version: z.enum(['v3', 'v4']).describe('The migration version (e.g., v4, v3)')
   },
+  inputExamples: [
+    { version: 'v4' }
+  ],
   cache: '30m',
   async handler({ version }) {
     const event = useEvent()
```

ESLint

[error] 'useEvent' is not defined.

(no-undef)